modules/nf-core/integronfinder/tests/main.nf.test

```diff
@@ -0,0 +1,64 @@
+nextflow_process {
+
+    name "Test Process INTEGRONFINDER"
+    script "../main.nf"
+    process "INTEGRONFINDER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "integronfinder"
+
+    test("IntegronFinder - escherichia_coli - genome_fa_gz") {
+
+        when {
+            process {
+                """                
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/prokaryotes/escherichia_coli/genome/genome.fa.gz', checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                        process.out.gbk ? path(process.out.gbk[0][1]).text : "no_gbk",
+                        process.out.integrons,
+                        process.out.summary,
+                        path(process.out.out[0]).readLines().get(56).contains("command used: integron_finder"),
```

Comment:
This part should be asserted outside the snapshot.
As it will only give true or false.